Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -380,6 +380,14 @@ prevent a NodePort service from hijacking traffic of a host local application
 which source port matches the service port. To disable the modification of
 the reserved ports, set ``global.nodePort.autoProtectPortRanges`` to ``false``.
 
+By default, the NodePort implementation prevents application ``bind(2)`` requests
+to NodePort service ports. In such case, the application will typically see a
+``bind: Operation not permitted`` error. This happens either globally for older
+kernels or starting from v5.7 kernels only for the host namespace by default
+and therefore not affecting any application Pod bind(2) requests anymore. In
```

Comment:
Nit:
```suggestion
and therefore not affecting any application pod ``bind(2)`` requests anymore. In
```

Reply:
Isn't `Pod` its own name and therefore written in capital letters (like Cilium or Kubernetes)? Wrt `bind(2)`, I only wanted to emphasize it in the first mention. But once GKE CI is green I can make that small amendment, rerun test-docs-please and then merge it, np.

Reply:
Yes, sorry. You're right regarding `Pod` being its own name. Though, it seems to be used in both upper- and lower case in our docs. Regarding `bind(2)` I thought it might be better to be consistent.

Reply:
Yeah I did check on Pod afterwards and majority of cases in the kube-proxy free guide had it as `pod`, so made it consistent. :-)